TodoApp/ViewController.swift

```diff
@@ -12,7 +12,14 @@ class ViewController: UIViewController {
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        var optional: String?
         // Do any additional setup after loading the view.
+
+        optional = "toto"
+
+        let tata = optional!
+
+        print(tata)
     }
 
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/coutar-t/Bitrise-CI/blob/542b9b0a748dff3bd6f40f08fe5332fde2be59a7/TodoApp/ViewController.swift#L25"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>ViewController.swift:25</code></td>
    </tr>
  </tbody>
</table>